README.md

```diff
@@ -69,10 +79,10 @@ A basic configuration looks like:
 default-host: dev-app.conduce.com
 default-user: enterprise_test@conduce.com
 dhl-dev@conduce.com:
-  api-keys:
```

Comment:
was this intended?  It seems like the map of env to api key should be under the api-keys section